contracts/keep3r/GenericKeep3rV2.sol

```diff
@@ -19,12 +20,16 @@ contract GenericKeep3rV2 is Governable, CollectableDust, Keep3r, IStrategyKeep3r
     EnumerableSet.AddressSet internal availableStrategies;
     mapping(address => uint256) public requiredHarvest;
```

Comment:
So the work flow for adding a strategy will involve asking the governor to set this maximum gas cost when they add a strategy for you?

What is the recommended way for strategy creators to find this maximum gas cost? Estimate gas limit? Or by running a harvest and reporting the gas used?

Reply:
Great questions. I am looking at my previous `harvest()` to estimate the max gas cost. 
I am not sure if there is a better way.